src/options-and-use.md

```diff
@@ -296,7 +296,7 @@ quadrilateral into the unit square:
 
 The above plot was produced with:
 ```sh
-glvis -m quad.vtk -g quad-vec.gf -k "fRjlAmeIIiiiiiiiiiiibbvuuuuuuuuuuu"
+glvis -m quad.vtk -g quad-vec.gf -k "RjlAmeOOooooooooooobbvuuuuuuuuuuu************"
```

Comment:
Unrelated, but I noticed that 'u' and 'U' aren't documented in vsvector.cpp.

Reply:
Good catch, I'll add a fix in GLVis/glvis#130.

Reply:
See https://github.com/GLVis/glvis/pull/130/commits/dc01fe6d0c153f70cb232406a41a726365528e9d